pkg/odo/cli/component/common_link.go

```
@@ -301,9 +295,18 @@ func (o *commonLinkOptions) run() (err error) {
 			return
 		}
 
+		// convert service binding request into a ma[string]interface{} type so
+		// as to use it with dynamic client
+		serviceBindingMap := make(map[string]interface{})
+		intermediate, _ := json.Marshal(o.serviceBinding)
```

Why are we ignoring the error here?

---

Good catch, fixed